services/evaluator/app/infrastructure/llm_provider.py

```
@@ -0,0 +1,360 @@
+        # --> Esta rama usa una función async; acá la corrés con loop sync
+        try:
+            loop = asyncio.get_event_loop() # --> Obtiene loop actual
+        except RuntimeError:
+            loop = asyncio.new_event_loop() # --> Crea loop si no hay
+            asyncio.set_event_loop(loop)
+        
+        interview = loop.run_until_complete(load_interview_from_supabase(identifier)) # --> Await bloqueante
+        if interview:
+            return interview # --> Ya viene como Interview
+        else:
+            raise ValueError(f"No interview found in Supabase with ID: {identifier}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)